dbdiff/utils.py

```
@@ -154,26 +155,41 @@ def get_models_tables(models):
+    # For a plain function (Django 4.x), it needs an explicit first arg.
+    _needs_explicit_cls = not isinstance(_raw, (classmethod, staticmethod))
+    _original = TransactionTestCase._reset_sequences
 
+    def _sqlite_reset(db_name):
+        connection = connections[db_name]
         if connection.vendor != 'sqlite':
             return
-
         tables = get_models_tables(apps.get_models())
         statements = [
             "UPDATE SQLITE_SEQUENCE SET SEQ=0 WHERE NAME='%s';" % t
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)